src/Lokad.ILPack.csproj

```diff
@@ -23,6 +23,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="Extensions.cs~RF6a0154.TMP" />
```

Comment:
I'm not sure what this is ?

Reply:
This is something that VisualStudio introduced somehow. I'm removing that.